web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -1906,140 +1917,140 @@ class AppVersionHistory extends Component<Props, State> {
                           />
                         </div>
                       )}
-                      {!gitopsIsConnected && (
-                        <div className="TableDiff--Wrapper card-bg u-marginBottom--30">
-                          {/* DEV MODE: make this false */}
-                          {!this.props.outletContext.isEmbeddedCluster && (
-                            <div className="flex justifyContent--spaceBetween alignItems--center u-marginBottom--15">
-                              <p className="u-fontSize--normal u-fontWeight--medium u-textColor--info">
-                                {this.state.updatesAvailable
-                                  ? "New version available"
-                                  : ""}
-                              </p>
-                              <div className="flex alignItems--center">
+
+                      {!gitopsIsConnected &&
+                        (this.state.updatesAvailable ||
+                          (this.state.availableUpdates &&
```

Comment:
prevents this div from rendering when there aren't any versions 
![Version History embedded-cluster-smoke-test-staging-app Admin Console 2024-06-26 at 1 04 04 PM](https://github.com/replicatedhq/kots/assets/28071398/146542c3-4f29-4942-9da4-99d1b5318983)